examples/web-wagmi/src/publications/UseCreateComment.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  CollectPolicyType,
+  ContentFocus,
+  ProfileFieldsFragment,
+  ReferencePolicy,
+  useCreateComment,
+} from '@lens-protocol/react';
+import { useState } from 'react';
+
+import { LoginButton } from '../components/auth/LoginButton';
+import { WhenLoggedIn, WhenLoggedOut } from '../components/auth/auth';
+import { upload } from '../upload';
+
+type CreatePostProps = {
+  activeProfile: ProfileFieldsFragment;
+};
+
+export function CreateComment({ activeProfile }: CreatePostProps) {
+  const [publicationId, setPublicationId] = useState<string>('0x1b-0x0118');
+  const [content, setContent] = useState<string>('');
+
+  const { create, error, isPending } = useCreateComment({ profile: activeProfile, upload });
+
+  const submit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (content.length === 0) {
+      return;
+    }
+    await create({
+      publicationId,
+      content,
+      profileId: activeProfile.id,
+      contentFocus: ContentFocus.TEXT,
+      locale: 'en',
+      collect: {
+        type: CollectPolicyType.NO_COLLECT,
+      },
+      reference: ReferencePolicy.ANYBODY,
+    });
+
+    setContent('');
+  };
+
+  return (
```

Comment:
can we show somewhere a notification/toast/message in the example component when the comment was created successfully?

Reply:
This is an interesting topic. I created a separate ticket to discuss how to move this proposal: https://avara.height.app/lens-sdk/T-2943